analysis/common_lib/data_reader.py

```diff
@@ -88,4 +88,11 @@ def read_data(self, source, tidy = False):
                 #range, and suicide_rate to store the value
                 data = pd.melt(data, id_vars = ['country','sex'], var_name = 'age_range', value_name = 'suicide_rate')
         
-        return(data)
\ No newline at end of file
+        return(data)
+
+    def add_up_suicide_rate_all_ages(self, source) :
+        source["all_age"] = source["80_above"]+ source["70to79"] + source["60to69"]+ source["50to59"]+ source["40to49"]+ source["30to39"]+ source["20to29"] + source["10to19"]
```

Comment:
data_reader should serve for getting data only
how about move this code to data_cleaning.py and rename data_cleaning.py to etl.py
save the data with this new columns into processed_data folder ?

Reply:
Thanks, Ming for the suggestion.
Shifting this piece of code to ELT is better idea.